documentation/docs/02-template-syntax/02-basic-markup.md

```diff
@@ -138,3 +138,13 @@ Comments beginning with `svelte-ignore` disable warnings for the next block of m
 <!-- svelte-ignore a11y-autofocus -->
 <input bind:value={name} autofocus />
 ```
+
+You can document components with the `@component` tag. Component comments will appear when hovering over a component in your IDE using the [Svelte Language Server](https://github.com/sveltejs/language-tools).
+
+```svelte
+<!--
+@component
+This is a component description.
```

Comment:
I'm assuming you'd normally start it on the same line? That's what we do with JSDoc at least...
```suggestion
@component this is a component description
```